docs/getting-started/ers-req.asciidoc

```diff
@@ -0,0 +1,44 @@
+[[ers-requirements]]
+= Entity Risk Scoring prerequisites
+
+To use <<entity-risk-scoring, Entity Risk Scoring>>, your role must have certain cluster, index, and {kib} privileges. You also need the https://www.elastic.co/subscriptions[appropriate license].
```

Comment:
Can we say more about which license is required? It looks like we use similarly general phrasing on other prereq pages, but the linked subscriptions page doesn't seem to mention entity risk scoring.

Reply:
Updated in https://github.com/elastic/security-docs/pull/4162/commits/de68953d45ec7e789c178393858d3e08cca909af.